.github/workflows/node-test.yml

```diff
@@ -38,6 +38,36 @@ jobs:
       - run: npm i -g npm@9.5
       - run: npm ci
       - run: npm run lint:changed-files
+      - run: npm run lint
+        working-directory: firebase-vscode
+
+  vscode_unit:
+    # Using windows to bypass an error thrown by VScode
+    # when run in an environment that does not have a screen.
+    runs-on: windows-latest
+    strategy:
+      matrix:
+        node-version:
+          - "18"
+          - "20"
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: npm
+          cache-dependency-path: npm-shrinkwrap.json
+
+      - run: npm i -g npm@9.5
+      - run: npm ci
```

Comment:
Double checking - if you remove this line, do the tests still pass?

Reply:
I doubt it. We have to run `npm i` in both the root and the vscode folder.
The `npm i -g` above doesn't count as it only updates the npm version and nothing else.